simulation.py

```diff
@@ -78,20 +79,42 @@ def extract_data(raw_data, data_list, price_dataset, region):
             name : raw_data[region][name]
             for name in data_list if name in raw_data[region]}}
 
-def handle_result(
-        results, eval_results, alpha_path,
-        quiet_mode, output_path):
+def out_put_on_cmd(results, instruments):
```

Comment:
Can this be convert_to_dataframe?  It only converts `results` to format `pandas.DataFrame` rather than outputs anything.